plugin-template

```diff
@@ -48,6 +48,7 @@ DEFAULT_SETTINGS = {
     'redmine_project': None,
     'test_bindings': True,
     'test_cli': False,
+    'fips_test': True,
```

Comment:
Is True the proper default here? I know we want that on most plugins, but usually we add features disabled by default here.
(e.g. `redmine_project`, `test_cli`, `test_performance`, `test_released_plugin_with_next_pulpcore_release`, `test_s3`, ...)

edit: I see, that i :+1:d the idea to add it True... So don't see this as a blocker.

Reply:
I suggested true by default but i think you’re right.